runners/spark/src/main/java/org/apache/beam/runners/spark/io/ConsoleIO.java

```diff
@@ -31,17 +32,27 @@ private ConsoleIO() {
 
   /**
    * Write on the console.
+   *
+   * @see StreamingTransformTranslator#print()
    */
   public static final class Write {
 
     private Write() {
     }
 
-    public static <T> Unbound<T> out() {
-      return new Unbound<>(10);
+    private static final Integer DEFAULT_NUM = 10;
```

Comment:
It would be nice to have all the fields positioned before any of the methods (ctor included).